lms/templates/navigation/navbar-not-authenticated.html

```diff
@@ -51,11 +51,11 @@
     <li class="item nav-courseware-01">
       % if not settings.FEATURES['DISABLE_LOGIN_BUTTON'] and not combined_login_and_register:
         % if course and settings.FEATURES.get('RESTRICT_ENROLL_BY_REG_METHOD') and course.enrollment_domain:
-          <a class="btn btn-login" href="${reverse('course-specific-login', args=[course.id.to_deprecated_string()])}${login_query()}">${_("Sign in")}</a>
+          <a class="btn btn-brand btn-login" href="${reverse('course-specific-login', args=[course.id.to_deprecated_string()])}${login_query()}">${_("Sign in")}</a>
```

Comment:
This PR was the first time that a pattern library page could be rendered for an anonymous user, so I had to make a number of changes to the header. This one was necessary to make the button render as a blue 'brand' button, rather than a normal button with a white background.